### PR TITLE
new metadata connection in hotelx content

### PR DESCRIPTION
**Pull request by zechao**
Hi @arquio 
This new service need you review.
We added the adviseMessage at connection level, to return errors that are not related to each metadata node, such as the validation error, connection error, etc.

### CODE REVIEW

**Comment by zechao** [comment]
Hi,
@arquio  need you review,

File: hotelx/hotstc/enums/enum_OptionType.graphql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Indicates the type of destination
+enum OptionType {
```
[the remark this answers is not in the document]
Comment (arquio) [reply hotelx/hotstc/enums/enum_OptionType.graphql]: HotelOpitionType

File: hotelx/hotstc/enums/enum_PaymentCardType.graphql [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# List of payment card supported
+enum PaymentCardType {
```
Comment (arquio) [inline hotelx/hotstc/enums/enum_PaymentCardType.graphql]: Exists

File: hotelx/hotstc/enums/enum_PaymentType.graphql [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Options payment type accepted by supplier
+enum PaymentType {
```
Comment (arquio) [inline hotelx/hotstc/enums/enum_PaymentType.graphql]: Exists

File: hotelx/hotstc/enums/enum_PreferenceType.graphql [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Type of preference allowed
+enum PreferenceType {
```
Comment (arquio) [inline hotelx/hotstc/enums/enum_PreferenceType.graphql]: BookPreferenceType?

File: hotelx/hotstc/enums/enum_RateRulesType.graphql [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+  LARGE_FAMILY
+
+  HONEYMOON
+
+  PUBLIC_SERVANT
+
+  UNEMPLOYED
+
+  MOBILE
```
Comment (arquio) [inline hotelx/hotstc/enums/enum_RateRulesType.graphql]: Exist , add

File: hotelx/hotstc/objects/object_MetadataAge.graphql [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Contain age range of the passenger
+type Age{
```
Comment (arquio) [inline hotelx/hotstc/objects/object_MetadataAge.graphql]: AgeRange

File: hotelx/hotstc/objects/object_MetadataAgeRange.graphql [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# Contain age range information and the review date
+type AgeRange{
```
Comment (arquio) [inline hotelx/hotstc/objects/object_MetadataAgeRange.graphql]: reviewedAgeRange

File: hotelx/hotstc/objects/object_MetadataBook.graphql [hunk 8]
```diff
@@ -0,0 +1,25 @@
+type Book{
```
Comment (arquio) [inline hotelx/hotstc/objects/object_MetadataBook.graphql]: MetadataBook

File: hotelx/hotstc/objects/object_MetadataBooking.graphql [hunk 9]
```diff
@@ -0,0 +1,21 @@
+# Return information about booking
+type Booking{
```
Comment (arquio) [inline hotelx/hotstc/objects/object_MetadataBooking.graphql]: MetadataBooking

File: hotelx/hotstc/objects/object_MetadataStaticInfo.graphql [hunk 10]
```diff
@@ -0,0 +1,7 @@
+# Informs the static type.The possible values are: Offline, OfflineCompleted or NotImplemented.
+type StaticInfo{
```
Comment (arquio) [inline hotelx/hotstc/objects/object_MetadataStaticInfo.graphql]: HotelStaticInfo

File: hotelx/hotstc/objects/object_MetadataStaticTypeInfo.graphql [hunk 11]
```diff
@@ -0,0 +1,7 @@
+# Informs the static type.The possible values are: Offline, OfflineCompleted or NotImplemented.
+type StaticTypeInfo{
```
Comment (arquio) [inline hotelx/hotstc/objects/object_MetadataStaticTypeInfo.graphql]: reviewedBol??

File: hotelx/hotstc/objects/object_MetadataStaticInfo.graphql (hunk 10 above)
Comment (zechao) [reply hotelx/hotstc/objects/object_MetadataStaticInfo.graphql]: changed to MetadataStatic

**Comment by zechao** [comment]
I fixed all requested changes and removed duplicate files

**Pull request merged**